packages/database/schema.prisma

```diff
@@ -34,6 +34,7 @@ model Webhook {
   createdAt     DateTime           @default(now()) @map(name: "created_at")
   updatedAt     DateTime           @updatedAt @map(name: "updated_at")
   url           String
+  source        String?            @default("user")
```

Comment:
Also `source` is optional (`String?`). I think we can leave out the question mark as we also set the default value for all already existing webhooks and new webhooks.